code/go/0chain.net/blobbercore/handler/handler_test.go

```diff
@@ -1610,12 +1599,7 @@ func TestHandlers_Requiring_Signature(t *testing.T) {
 					}
 					rmData, err := json.Marshal(rm)
 					require.NoError(t, err)
-					require.NoError(t, formWriter.WriteField("read_marker", string(rmData)))
-					if err := formWriter.Close(); err != nil {
-						t.Fatal(err)
-					}
-					r, err := http.NewRequest(http.MethodPost, url.String(), body)
-					r.Header.Add("Content-Type", formWriter.FormDataContentType())
+					r, err := http.NewRequest(http.MethodPost, url.String(), nil)
```

Comment:
http.MethodPost should be updated with http.MethodGet. and please move all download tests to hanlder_download_test.go.  There are too many tests on handler_test.go. it is very difficult to read and debug.